.circleci/config.yml

```diff
@@ -48,9 +48,9 @@ commands:
       force_native_shadow_mode:
         type: boolean
         default: false
-      enable_native_custom_element_lifecycle:
+      api_version_59:
         type: boolean
-        default: false
+        default: true
```

Comment:
I would love to use the `integer` type here, but I could not get it to work, and I figured there's little point in debugging Circle CI oddities since we may move away from Circle CI anyway.